src/main/java/MarkNothingException.java

```diff
@@ -0,0 +1,5 @@
+public class MarkNothingException extends Exception {
```

Comment:
Perhaps there could be a better name for this exception? perhaps InvalidTaskException or something 😬

Reply:
It's a good suggestion. I already name it InvalidTaskException. Thanks.